geometry_msgs/Pose currentPos
geometry_msgs/Pose nextPos
---
string action